fmx_annealer.py

```diff
@@ -5,6 +5,7 @@
 import time
 
 import socket
```

Comment:
no longer necessary now that you aren't checking hostname

Reply:
Agreed